pkg/v16/resource/certconfig/desired.go

```diff
@@ -238,9 +238,10 @@ func newOperatorAPICertConfig(clusterConfig cluster.Config, cert certs.Cert, nam
 			Name:      key.CertConfigName(clusterConfig.ClusterID, cert),
 			Namespace: namespace,
 			Labels: map[string]string{
-				label.Cluster:      clusterConfig.ClusterID,
-				label.ManagedBy:    projectName,
-				label.Organization: clusterConfig.Organization,
+				label.Cluster:         clusterConfig.ClusterID,
+				label.LegacyClusterID: clusterConfig.ClusterID,
```

Comment:
Well, this is backwards. We must not use the legacy label anymore. That is why it is called legacy. We need a solution without it.

Reply:
Well, how about then create all `CertConfig` with `ClusterID` and search for current CRs with `ClusterID`?

Reply:
Do you see any possible problem with the above suggestion?

Reply:
Yes agree with @xh3b4sd . We need to use the new label which is `giantswarm.io/cluster`. We should be able to update the existing certconfig CRs to have this label.

See https://github.com/giantswarm/fmt/blob/master/kubernetes/annotations_and_labels.md